src/PrestaShopBundle/Utils/Migrate.php

```diff
@@ -39,10 +46,15 @@ public static function migrateSettingsFile(Event $event = null)
 
                 $default_parameters = Yaml::parse($root_dir.'/app/config/parameters.yml.dist');
 
+                if (defined('_LEGACY_RIJNDAEL_KEY_')) {
+                    $default_parameters['parameters']['_rijndael_key'] = _LEGACY_RIJNDAEL_KEY_;
+                    $default_parameters['parameters']['_rijndael_iv'] = _LEGACY_RIJNDAEL_IV_;
+                }
+
                 $parameters = array(
                     'parameters' => array(
                         'database_host'     => _LEGACY_DB_SERVER_,
-                        'database_port'     => '~',
+                        'database_port'     => '3306',
```

Comment:
With my local db config, it fails with '~' :
<?php
define('_DB_SERVER_', 'localhost');
define('_DB_NAME_', 'prestashop');
define('_DB_USER_', 'root');
define('_DB_PASSWD_', ''); 

=> 

Init CLDR data download...
Link to database cannot be established: SQLSTATE[HY000] [2002] Can't assign requested address